### ISSUE TITLE
508-defect-1 ❗ Launchblocker [SCREENREADER]: Screen reader users must be informed that the representative documentation section is incomplete

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#24500]
# [508-defect-1 :exclamation: Launchblocker](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh

## User Story or Problem Statement

As a screen reader user, I want to know if I must complete an additional section on the review page if I change a selection.

## Details

If a screen reader user decides to switch their choice to "Yes, I want to upload my document now," they are not provided any indication that it must be done in the following section as the visual alert icon does not have alt text AND no specific error or alert is announced.

## Acceptance Criteria

- [ ] Recommended, upon switching their selection to "Yes, I want to upload my document now" and clicking "update page" the user should receive a visual alert that announces the need of completing the following section.
- [ ] Band-aid, provide alternative text to the visual warning icons using `aria-label`.
-->

## Steps to Recreate

1. Select any choice but "yes, I want to upload my document now" and reach the review page
2. On the review page using a screen reader, change your choice to "yes, I want to upload my document now"
3. Confirm that visually alert icons appear, but the screen reader announces nothing
4. Confirm that you are unable to access the alert icons (since they have the `role="presentation"`)

### Recommended, provide a visible alert informing users that they must complete the following section.
Provide an error message or alert informing the user that they must complete the "upload your legal documentation section" upon pressing the "update page" button for representative documentation.
<img width="1540" alt="Screen Shot 2021-05-10 at 11 40 42 PM" src="https://user-images.githubusercontent.com/14154792/117754892-3c2cd700-b1e9-11eb-99ff-b3f2352dbd80.png">

### Bandaid, provide alt text for the warning icon and use `aria-describedby` to link it to the button heading
<img width="1591" alt="Screen Shot 2021-05-10 at 11 31 27 PM" src="https://user-images.githubusercontent.com/14154792/117754318-18b55c80-b1e8-11eb-8422-ec634ab6a79e.png">

## WCAG or Vendor Guidance (optional)
* [WCAG 3.3.1 – Error Identification (Level A)](https://www.wuhcag.com/error-identification/)

## Screenshots or Trace Logs
<img width="1530" alt="Screen Shot 2021-05-10 at 11 05 45 PM" src="https://user-images.githubusercontent.com/14154792/117754975-67172b00-b1e9-11eb-9d73-612b5677cf97.png">
<img width="800" alt="Screen Shot 2021-05-10 at 10 56 28 PM" src="https://user-images.githubusercontent.com/14154792/117754992-6bdbdf00-b1e9-11eb-92ab-7895801d5330.png">

**Comment by caw310** [comment]
Seems like this should be fixed. Needs prioritization.
